src/lib/net/utilities/remote.ts

```diff
@@ -0,0 +1,41 @@
+import { RunService } from "@rbxts/services";
+import { ClientNet } from "./client";
+import { Identifiers } from "./identifiers";
+import { ServerNet } from "./server";
+
+const isServer = RunService.IsServer();
+
+export class remote {
```

Comment:
Why does this co-exist with `RemoteClass`? Try to keep it to just different types of `Endpoint`s for different methods.

Reply:
I have it for reference, will be deleted next push.